src/proposals/mips/mip-b17/mip-b17.sol

```diff
@@ -0,0 +1,589 @@
+                                abi.encodePacked(
+                                    "JUMP_RATE_IRM_",
+                                    config.addressesString
+                                )
+                            )
+                        ),
+                        unitroller: addresses.getAddress("UNITROLLER")
+                    });
+
+                    /// calculate initial exchange rate
+                    /// BigNumber.from("10").pow(token.decimals + 8).mul("2");
+                    /// (10 ** (18 + 8)) * 2 // 18 decimals example
+                    ///    = 2e26
+                    /// (10 ** (6 + 8)) * 2 // 6 decimals example
+                    ///    = 2e14
+                    uint256 initialExchangeRate = (10 **
+                        (ERC20(addresses.getAddress(config.tokenAddressName))
+                            .decimals() + 8)) * 2;
+
+                    MErc20Delegator mToken = new MErc20Delegator(
+                        addresses.getAddress(config.tokenAddressName),
+                        ComptrollerInterface(addr.unitroller),
+                        InterestRateModel(addr.irModel),
+                        initialExchangeRate,
+                        config.name,
+                        config.symbol,
+                        mTokenDecimals,
+                        payable(deployer),
+                        addr.mTokenImpl,
+                        ""
+                    );
+
+                    addresses.addAddress(
+                        config.addressesString,
+                        address(mToken),
+                        true
+                    );
+                }
+            }
+        }
+    }
+
+    function afterDeploy(Addresses addresses, address) public override {
+        address governor = addresses.getAddress("TEMPORAL_GOVERNOR");
+        Configs.CTokenConfiguration[]
+            memory cTokenConfigs = getCTokenConfigurations(block.chainid);
+
+        unchecked {
+            for (uint256 i = 0; i < cTokenConfigs.length; i++) {
+                Configs.CTokenConfiguration memory config = cTokenConfigs[i];
+                supplyCaps.push(config.supplyCap);
+                borrowCaps.push(config.borrowCap);
+
+                /// get the mToken
+                mTokens.push(
+                    MToken(addresses.getAddress(config.addressesString))
+                );
+
+                _validateCaps(addresses, config); /// validate supply and borrow caps
+
+                if (
+                    mTokens[i].reserveFactorMantissa() !=
+                    config.reserveFactor &&
+                    mTokens[i].protocolSeizeShareMantissa() != config.seizeShare
+                ) {
+                    mTokens[i]._setReserveFactor(config.reserveFactor);
+                    mTokens[i]._setProtocolSeizeShare(config.seizeShare);
+                    mTokens[i]._setPendingAdmin(payable(governor)); /// set governor as pending admin of the mToken
```

Comment:
do we need this?

Reply:
Shouldn't we include this outside the if statement? Even if the reserveFactor and seizeShare are accurate, we still need to set the pending admin, right?